serveradmin/serverdb/querybuilder.py

```diff
@@ -102,29 +102,7 @@ def get_sql_condition(self, attribute, filt):  # NOQA C901
                 ))
 
         elif isinstance(filt, Overlaps):
-            if attribute.type != 'inet':
-                raise FilterValueError(
-                    'Cannot network filter attribute "{}"'.format(attribute)
-                )
-
-            elif isinstance(filt, ContainedOnlyBy):
-                operator = '<<'
-            elif isinstance(filt, InsideNetwork):
-                operator = '<<='
-            else:
-                operator = '&&'
-
-            template = "{{}} {} '{}'".format(operator, filt.value)
-            if isinstance(filt, ContainedOnlyBy):
-                template += (
-                    ' AND NOT EXISTS ('
-                    '   SELECT 1 '
-                    '   FROM server AS supernet '
-                    '   WHERE {{}} << supernet.intern_ip AND '
-                    '       supernet.intern_ip << {}'
-                    ')'
-                    .format(filt.value)
-                )
+            template = self._containment_filter_template(attribute, filt)
```

Comment:
I know its just a separation, however, theoretically it does not fit with the commit message. 😜

Reply:
Why not?

Reply:
Because the commit message just says about a Renaming of a class and not a separation and therefore addition of a method. However, I don't care about that here. Was just mentioning it. 😉